app/javascript/components/map/MapPopup.vue

```diff
@@ -110,6 +100,16 @@ export default {
       this.randomKey += 1;
       this.data.datasets = [];
 
+      const ids =[]
+      
+      this.chartData.forEach(item => ids.push(item.id))
```

Comment:
I think this is redundant now, but see the comment below - I think it can be simplified even more